docs_sources/using.md

```diff
@@ -457,6 +457,16 @@ Currently only classification is supported, but future releases will include reg
 <td align="center"><a href="https://github.com/EpistasisLab/tpot/blob/master/tpot/config/classifier_nn.py">Classification</a></td>
 </tr>
 
+<tr>
+<td>TPOT-cuML</td>
+<td>TPOT will search over a restricted configuration using the GPU-accelerated estimators in <a href="https://github.com/rapidsai/cuml">RAPIDS cuML</a> and <a href="https://github.com/dmlc/xgboost">DMLC XGBoost</a>. This configuration requires an NVIDIA Pascal architecture or better GPU with compute capability 6.0+, and that the library cuML is installed. With this configuration, all model training and predicting will be GPU-accelerated.
+<br /><br />
+This configuration is particularly useful for medium-sized and larger datasets on which CPU-based estimators are a common bottleneck, and works for both the TPOTClassifier and TPOTRegressor.</td>
+<td align="center"><a href="https://github.com/EpistasisLab/tpot/blob/master/tpot/config/classifier_cuml.py">Classification</a>
```

Comment:
@weixuanfu for these documentation hyperlinks, should I be hard-coding the `master` branch like the existing documentation? Or, since this PR is targeting the development branch, should these refer to `development

Reply:
No need that, eventually it will be merged to master branch.